src/os/text.js

```diff
@@ -0,0 +1,22 @@
+goog.module('os.Text');
+
+/**
+ * Simple object to handle logical grouping of displayed text in Opensphere
+ */
+class Text {
+  /**
+   * Constructor for a Text
+   * @param {string=} id a key, CSS class, etc to help find the element
+   * @param {string=} label the visible string
+   * @param {string=} tooltip the string shown on hover
+   * @param {string=} description a long explanation of the button or feature
+   */
+  constructor(id = '0', label = '', tooltip = '', description = '') {
+    this.id = id;
```

Comment:
I don't think this is a good association of data. If you want constants for strings so they are reusable, a simple enum will suffice:
```
/**
 * @enum {string}
 */
const StringConstants = {
  MENU_SELECTED_LABEL: 'blah',
  MENU_SELECTED_TOOLTIP: 'blah blah',
  MENU_SELECTED_DESCRIPTION: 'blah blah blah',
};
```
Then this is more reusable for other items too, which may not have ids, tooltips, or descriptions.